src/sage/schemes/curves/zariski_vankampen.py

```diff
@@ -827,7 +849,7 @@ def braid_in_segment(glist, x0, x1, precision={}):
 
     Check that :trac:`26503` is fixed::
 
-        sage: # needs sage.rings.real_mpfr sage.symbolic
+        sage: # needs sage.rings.real_mpfr sage.symbolic sirocco
```

Comment:
Only one test in here requires sirocco, so you should leave it on that test (otherwise the whole block won't get tested unless sirocco is installed!).

Reply:
You are right, but the test has sense only to prepair the last order.

Reply:
Yes, but that means all of those preparatory tests are not done by default, which means they aren't fixed if something changes elsewhere (and also means it becomes more likely bugs are introduced). It also tells the user exactly what method requires `sirocco`. These block test markers should only be added when it applies to most of the tests (such as stemming from the first one).

This same comment and reasoning applies to many other doctests and changes.

Reply:
Changed at several places